zapisy/apps/users/views.py

```diff
@@ -23,7 +24,7 @@
 
 @login_required
 @external_contractor_forbidden
-def students_view(request, user_id: int = None):
+def students_view(request, user_id: int = None, semester_id: Optional[int] = None):
```

Comment:
Skoro już dotykamy tego wiersza, to czy taka anotacja typu nie byłaby lepsza (także w drugiej analogicznej funkcji)?
```suggestion
def students_view(request, user_id: Optional[int] = None, semester_id: Optional[int] = None):
```
(... i czy to w ogóle jest sprawdzane, skoro do tej pory też działało?!)

Reply:
Z tego co wiem typehinty nie są bezpośrednio sprawdzane przez Python, zatem jeśli typ argumentu user_id był określony jako int, to Python nie robił problemu przy przypisaniu mu wartości None. Dopiero zastosowanie narzędzi takich jak np mypy mogłoby tą zgodność typów wymuszać.